typescript/src/adjustAutoImports.ts

```diff
@@ -66,7 +66,7 @@ export function parseIgnoreSpec(spec: string): ParsedIgnoreSetting {
 }
 
 export const getIgnoreAutoImportSetting = (c: GetConfig) => {
-    return c('suggestions.ignoreAutoImports').map(spec => parseIgnoreSpec(spec))
+    return c('suggestions.ignoreAutoImports')?.map(spec => parseIgnoreSpec(spec))
```

Comment:
Why is this needed? You should not get crash as all config should be passed from volarConfig. Maybe you had this extension disabled?